version.txt

```diff
@@ -1,6 +1,6 @@
 <version>
-<lastupdateYYMMDD>201025</lastupdateYYMMDD>
+<lastupdateYYMMDD>220302</lastupdateYYMMDD>
 <versionurl>https://raw.githubusercontent.com/NoSoup4you/Shadowfist-LackeyCCG/master/version.txt</versionurl>
 <updateurl>https://raw.githubusercontent.com/NoSoup4you/Shadowfist-LackeyCCG/master/updatelist.txt</updateurl>
-<message>Join the Discord: https://discord.gg/9ndyeRF</message>
+<message>Update all cards text with most current errata.</message>
```

Comment:
I can put this message back if you prefer (I only just saw the original message) but I'll have to generate a new checksum to do it. Not a big deal.